ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -266,14 +267,19 @@ export const ConceptHomepage = withCurrentWorkspace()(
       }
     }
 
-    triggerSearch(e) {
+    handleSearchStringChange(e) {
+      this.setState({currentSearchString: e});
+    }
+
+    searchOnEnterKeyPress(e) {
```

Comment:
Nitpick redux: maybe this contradicts my previous comments, but this name reads somewhat awkwardly now. Maybe prefer consistency with the above method, for something like "handleSearchKeyPress"?